doc/kernel/threads/scheduling.rst

```diff
@@ -175,6 +175,39 @@ becomes the current thread, its non-preemptible status is maintained.
     Locking out the scheduler is a more efficient way for a preemptible thread
     to inhibit preemption than changing its priority level to a negative value.
 
+.. _metairq_priorities:
+
+Meta-IRQ Priorities
+===================
+
+When enabled (see CONFIG_NUM_METAIRQ_PRIORITIES), there is a special
```

Comment:
add a link to the option by writing this as:

    When enabled (see :option:`CONFIG_NUM_METAIRQ_PRIORITIES`), ...